server/controllers/stock/lots.js

```diff
@@ -442,3 +455,65 @@ function assignments(req, res, next) {
     .catch(next)
     .done();
 }
+
+/**
+ * GET /lots/generate_barcodes/:number
+ *
+ * @description
+ * Returns generated barcodes in a zip file
+ */
+async function generateBarcodes(req, res, next) {
+
+  try {
+    const totalBarcodes = req.params.number;
+    const { key } = identifiers.LOT;
```

Comment:
We should probably add the ability to send our own identifier from the client to make this more flexible.